CHANGELOG.md

```diff
@@ -8,9 +8,10 @@
 * Changed the support version range of the `analyzer_plugin` to `>=0.3.0 <0.5.0`
```

Comment:
@incendial please remove or update to ^0.5.0

Reply:
Let's do that in the null safety migration request